.env.template

```
@@ -1,64 +1,70 @@
-── API Configuration ───────────────────────────────────────
-API_KEY=dev-key-sha256-not-for-production
-LOG_LEVEL=debug
-CORS_ORIGINS=["http://localhost:3000","http://localhost:8000"]
+# --- GDS ---
+GDS_ENABLED=true
+GDS_MAX_STALENESS_HOURS=25
 
-── Scoring Weights ─────────────────────────────────────────
+# --- Tenant Auth ---
+TENANT_AUTH_ENABLED=true
+TENANT_AUTH_BYPASS_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TENANT_AUTH_BYPASS_KEY key should go before the TENANT_AUTH_ENABLED key

(UnorderedKey)